simul/platform/deterlab.go

```diff
@@ -74,43 +79,32 @@ type Deterlab struct {
 	// Port number of the monitor and the proxy
 	MonitorPort int
 
-	// Which app to run
-	App string
 	// Number of machines
-	Machines int
-	// Processes Per Machines
-	Ppm int
-	// Number of Rounds
-	Rounds int
+	Hosts int
 	// Channel to communication stopping of experiment
 	sshDeter chan string
 	// Whether the simulation is started
 	started bool
 	// Debugging-level: 0 is none - 5 is everything
 	Debug int
-
-	// All hostnames used concatenated with the port
-	Hostnames []string
-
-	// Testing the connection?
-	TestConnect bool
 }
 
+var simulConfig *sda.SimulationConfig
+
 func (d *Deterlab) Configure() {
 	// Directory setup - would also be possible in /tmp
 	pwd, _ := os.Getwd()
+	d.CothorityDir = pwd + "/cothority"
 	d.DeterDir = pwd + "/platform/deterlab"
```

Comment:
Le 03/02/2016 15:52, Ismail a écrit :
>
> In simul/platform/deterlab.go 
> <https://github.com/dedis/cothority/pull/186#discussion_r51729963>:
>
> >  func (d *Deterlab) Configure() {
> >  	// Directory setup - would also be possible in /tmp
> >  	pwd, _ := os.Getwd()
> > +	d.CothorityDir = pwd + "/cothority"
> >  	d.DeterDir = pwd + "/platform/deterlab"
>
> Insn't this constant, too?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/dedis/cothority/pull/186/files#r51729963>.
>
the `pwd`-part is not constant, this is why I create these here.